### DISCUSSION TITLE
Unexpected error in Achievements

### DISCUSSION THREAD

**Discussion by X-itachi-X** [lowlighter/metrics#1483] [Q&A] [answered]
I have been using it for a month or two but now the achievements is showing me unexpected error and I have no idea why is this showing.

![image](https://github.com/lowlighter/metrics/assets/104882734/d7d38ecf-0d1e-4749-b38c-3e503db6a062)

**Comment by Ankitz007** [comment, marked as answer]
Hi. Please replace the following in your action workflow file for achievements workflow -
` - uses: lowlighter/metrics@latest`

to this - 
`- uses: lowlighter/metrics@96dde98a034ab75fec7884f9161bc12759275d36`

FYI, it's the hash for the commit after which @lowlighter merged the workaround after recent changes in github search. (#1487)

**Reply by Ankitz007** [reply]
A new release will fix the issues. No need to use commit hash, use latest version instead.

Regards,
Ankit
